docs/system/container-metrics.md

```diff
@@ -0,0 +1,104 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## Container Metrics
+
+### Metric: `container.cpu.time`
+
+This metric is optional.
```

Comment:
The metric requirement level is a standard thig defined here - https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/metric-requirement-level.md. Could you please use the corresponding value and add a link to the spec?

Same for other metrics in this doc